### PR TITLE
Travis: tweak how coverage is collected

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: etc/gather_coverage.jl [hunk 1]
```diff
@@ -6,8 +6,9 @@ using Coverage
 import JSON
 
 # gather coverage
-data = Codecov.process_folder(pwd())
-data_dict = Codecov.to_json(data)
+data = process_folder("src")
+data = append!(data, process_folder("pkg"))
```
Comment (fingolfin) [inline etc/gather_coverage.jl]: Hrm, I must be missing something -- for some reason, this collects data in `pkg`, but fails to collect it in `src`...

Comment (sebasguts) [reply etc/gather_coverage.jl]: I am not sure what you mean. In this branch, `src` only has `GAP.jl`, which gets collected. Also, the `.c` files get collected. So I guess it is fine.

**Comment by sebasguts** [comment]
Feel free to merge it once you agree with my comment.

File: etc/gather_coverage.jl (hunk 1 above)
Comment (fingolfin) [reply etc/gather_coverage.jl]: Err, no, `src/GAP.jl` does *not* get collected, as the codecov report clearly showed, and as now also https://codecov.io/gh/oscar-system/GAP.jl shows. So merging this unfortunately was incorrect. I'll revert it.

**Comment by fingolfin** [comment]
@mohamed-barakat I am afraid that merge was a bit premature.

**Comment by mohamed-barakat** [comment]
I am sorry, I oversaw the file. Please feel free to revert.

**Comment by sebasguts** [comment]
Yeah, I don't know what I checked . . .

**Pull request merged**